### PR TITLE
Developer/nikolaj k/cleanup

### PR DESCRIPTION
**Pull request by Nikolaj-K**
Readme with pictures and a bit formatting

### CODE REVIEW

**Comment by Nikolaj-K** [comment]
Should I do another pull request with only the changes to the readme? 
I think the code folder tree and the screenshoud images I had made should make it in.

**Comment by metachris** [comment]
I think multiple small and specific pull requests are better. It makes it easier to understand what is going on and have separate discussions about the specific changes. In this PR there is a lot of changes too.

Just a small note: Typical Python code conventions (PEP8) are one blank line between class methods, two blank lines only between class definitions and module-level methods. This pull request adds several extra blank lines between class methods.

**Comment by Nikolaj-K** [comment]
Thanks for the input - I was under the impression I just following PyCharms conventions.

File: README.md [hunk 1]
```diff
@@ -39,47 +39,79 @@
...
+    ├── Network
+        └── Payloads
+    ├── SmartContract
+        ├── Framework
+            ├── Neo
+            └── System
+    ├── Utils
+    ├── VM
+    └── Wallets
+```
```
Comment (localhuman) [inline README.md]: I would rather have a docs setup similar to neo-boa to put more extended explanations like this.

File: README.md [hunk 2]
```diff
@@ -120,43 +151,16 @@ installation command
...
-
-neo> show state
-Progress: 1054913 / 1237188
+After installing requirements and activating your environment, 
+there is an easy to use `prompt.py` file for you to run the node 
+as well as some basic interactivity:
 
-neo>
-```
+![opening prompt.py](http://i.imgur.com/jQklFSB.png)
```
Comment (localhuman) [inline README.md]: Same issue as above, would like to keep the main readme as simple as possible, and put more information in a docs build.

**Comment by localhuman** [comment]
Sorry about getting around to this so late.  Because so much time has passed, this PR will need to be reworked.   
Could you resubmit any changes that are still relevant to the `development` branch?  Thanks~

**Pull request closed without merging**